etc/adminhtml/system.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" ?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <section id="system" translate="label" type="text" sortOrder="900" showInDefault="1">
+            <tab>advanced</tab>
+            <resource>Aligent_AsyncEvents::manage</resource>
+            <group id="async_events" translate="label" type="text" sortOrder="1000" showInDefault="1">
+                <label>Async Events</label>
+                <field id="retry" translate="label" type="text" sortOrder="10" showInDefault="1">
+                    <label>Retry Count</label>
```

Comment:
Just realised that we'll need to change this to something like `Death Count` or `Maximum Deaths`

Reply:
I will change it to Death Count

Reply:
I will change it to `Death Count` :+1:

Reply:
I changed it to max_deaths